ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -534,6 +578,40 @@ namespace OrdinalAxis {
             return groupIntervalFactor;
         }
 
+        /**
+         * Get index of point inside the ordinal positions array.
+         *
+         * @private
+         * @param {number} val
+         *        The pixel value of a point.
+         *
+         * @param {Array<number>} [ordinallArray]
+         *        An array of all points available on the axis
+         *        for the given data set.
+         *        Either ordinalPositions if the value is inside the plotArea
+         *        or extendedOrdinalPositions if not.
+         *
+         * @return {number}
+         */
+        public getIndexOfPoint(
+            val: number,
+            ordinalArray: Array<number>
+        ): number {
+            const ordinal = this,
+                axis = ordinal.axis,
+                firstPointVal = ordinal.positions ? ordinal.positions[0] : 0,
+                // toValue for the first point.
+                firstPointX = ordinal.slope ? ordinal.slope * axis.transA : 0;
```

Comment:
can `axis.transA` also be undefined?

Reply:
TS said that it should exist and I believe him ;)